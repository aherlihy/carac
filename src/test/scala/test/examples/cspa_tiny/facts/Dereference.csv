String	String
0	1
2	3
4	5
3	6
7	0
8	9
10	4
11	8
12	13
14	15
13	16
17	14
19	20
21	22
23	24
25	26
27	28
30	27
22	31
20	18
32	33
26	29
34	23
33	35
36	37
37	38
39	40
41	42
43	44
45	46
47	48
49	50
51	52
53	54
56	57
59	60
61	62
63	64
66	67
68	69
71	72
75	76
77	78
79	80
83	84
89	55
97	98
99	58
103	104
105	106
108	70
