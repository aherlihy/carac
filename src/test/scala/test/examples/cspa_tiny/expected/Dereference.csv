Int	Int
63	64
56	57
66	67
68	69
103	104
12	13
27	28
20	18
97	98
79	80
59	60
7	0
33	35
19	20
23	24
108	70
8	9
26	29
83	84
99	58
49	50
47	48
2	3
34	23
45	46
39	40
4	5
11	8
37	38
53	54
61	62
13	16
77	78
25	26
30	27
41	42
43	44
71	72
3	6
36	37
14	15
51	52
22	31
105	106
89	55
17	14
75	76
32	33
0	1
10	4
21	22
